raft/node.go

```diff
@@ -22,6 +22,13 @@ import (
 	pb "github.com/coreos/etcd/raft/raftpb"
 )
 
+type SnapshotStatus int
+
+const (
+	SnapshotSuccess SnapshotStatus = 1
```

Comment:
i do not have this feeling at all...
but i would like to change to finish for consistency reason. we use snapshotFinish as the func name.